### PR TITLE
feat: Reduce DTS slice limit for regular messages on system subnets

### PR DESCRIPTION
**Pull request by dsarlis**
This is a follow-up change to further decrease the DTS slice limit on system subnets for regular messages. The value is set to 2B instructions to match the setting on application subnets.

`install_code` messages still use the bigger slice limit until the checkpoint interval is adjusted and we can afford reducing that as well. This will change in a future PR.

### CODE REVIEW

**Review by ulan — APPROVED** [review]
🚀

**Comment by schneiderstefan** [comment]
Are we doing this now despite delaying changing the checkpoint interval on the NNS subnet until after the scheduled II subnet recovery?

**Comment by dsarlis** [comment]
@schneiderstefan Yes, if you notice, I'm only changing the slice limit for regular messages. Given a 2B slice limit and the 50B total limit, we would have at most 25 slices. That's still quite fine even with a 200 rounds checkpoint interval.

Increasing the checkpoint interval blocks decreasing the slice limit for install code messages. With current 10B slice and 1T total limit we have a maximum 100 slices for an install_code message, so further reduction would require first increasing the checkpoint interval.

File: rs/config/src/subnet_config.rs
Comment (schneiderstefan) [inline rs/config/src/subnet_config.rs:1]: Not sure why I can't add a comment on line 316 (old)/ 318 (new), but isn't that wrong now? That should be `max_instructions_per_install_code_slice,`.

Comment (dsarlis) [reply rs/config/src/subnet_config.rs:1]: Oh yeah, good catch. I thought I'd done this change too.

Comment (dsarlis) [reply rs/config/src/subnet_config.rs:1]: I guess we need some test that checks whether the "correct" number of slices were used (although that's what you get when you want to have different limits...)

**Comment by dsarlis** [comment]
Marked as `DO_NOT_MERGE` until the NNS team has re-run the tests we had run when DTS was first enabled on system subnets.

**Comment by dsarlis** [comment]
> Marked as `DO_NOT_MERGE` until the NNS team has re-run the tests we had run when DTS was first enabled on system subnets.

Tests were successful.

**Review by schneiderstefan — APPROVED** [review]

**Pull request merged**